test/functional/fixtures/request-pipeline/xhr/test.js

```diff
@@ -0,0 +1,18 @@
+const { errorInEachBrowserContains } = require('../../../assertion-helper.js');
+
+describe('XHR', () => {
+    it('Should keep header if request was reopened', () => {
+        return runTests('./testcafe-fixtures/index.js', 'Click test header button');
+    });
+
+    it('Should authorization prefix for authorization header', () => {
```

Comment:
Lost a verb in the test name - `Should <verb> authorization prefix for authorization header`